src/Admin/SiteHealth.php

```diff
@@ -171,6 +207,111 @@ public function persistent_object_cache() {
 		];
 	}
 
+	/**
+	 * Return list of available object cache mechanism.
+	 *
+	 * @return array
+	 */
+	public function check_available_cache() {
+		return [
+			'redis'     => $this->is_site_has_redis(),
+			'memcached' => $this->is_site_has_memcached(),
+			'apcu'      => $this->is_site_has_apcu(),
+		];
+	}
+
+	/**
+	 * Check if site has redis cache mechanism.
+	 *
+	 * @return bool True if redis cache is available otherwise False.
+	 */
+	public function is_site_has_redis() {
+
+		if ( ! class_exists( 'Redis' ) ) {
+			return false;
+		}
+
+		/*
+		 * Try to connect redis server.
+		 */
+		$redis_server = [
+			'host'           => isset( $_SERVER['CACHE_HOST'] ) ? sanitize_text_field( $_SERVER['CACHE_HOST'] ) : '127.0.0.1',
+			'port'           => isset( $_SERVER['CACHE_PORT'] ) ? intval( $_SERVER['CACHE_PORT'] ) : 6379,
```

Comment:
Type casting via `(int)` is marginally faster.

```suggestion
			'port'           => isset( $_SERVER['CACHE_PORT'] ) ? (int) $_SERVER['CACHE_PORT'] : 6379,
```